src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
+        for (int i = 0; i < machines.length;i++) {
```

Comment:
```suggestion
        for (int i = 0; i < machines.length; i++) {
```
also, you can use for each here